logfire/_internal/config.py

```diff
@@ -252,13 +253,24 @@ def configure(  # noqa: D417
     Args:
         local: If `True`, configures and returns a `Logfire` instance that is not the default global instance.
             Use this to create multiple separate configurations, e.g. to send to different projects.
-        send_to_logfire: Whether to send logs to logfire.dev. Defaults to the `LOGFIRE_SEND_TO_LOGFIRE` environment
-            variable if set, otherwise defaults to `True`. If `if-token-present` is provided, logs will only be sent if
-            a token is present.
-        token: The project token. Defaults to the `LOGFIRE_TOKEN` environment variable.
-        service_name: Name of this service. Defaults to the `LOGFIRE_SERVICE_NAME` environment variable.
-        service_version: Version of this service. Defaults to the `LOGFIRE_SERVICE_VERSION` environment variable, or the
-            current git commit hash if available.
+        send_to_logfire: Whether to send logs to logfire.dev.
+
+            Defaults to the `LOGFIRE_SEND_TO_LOGFIRE` environment variable if set, otherwise defaults to `True`.
+            If `if-token-present` is provided, logs will only be sent if a token is present.
+        token: The project token.
+
+            Defaults to the `LOGFIRE_TOKEN` environment variable.
+        service_name: Name of this service.
```

Comment:
```suggestion
            Defaults to the `LOGFIRE_TOKEN` environment variable.

        service_name: Name of this service.
```

can we do this to make it more readable? otherwise it's confusing which lines are grouped together.